vue/components/molecules/restrictions-alert/restrictions-alert.vue

```diff
@@ -77,14 +77,19 @@ export const RestrictionsAlert = {
             visible: false
         };
     },
-    mounted: function() {
-        this.$bus.bind("restrictions", (changes, newPart) => {
+    created: function() {
+        this.onRestrictions = this.$bus.bind("restrictions", (changes, newPart) => {
             this.visible = changes.length > 0;
         });
+
+        this.onUndo = this.$bus.bind("undo", () => this.close());
+    },
+    destroyed: function() {
+        if (this.onRestrictions) this.$bus.unbind("restrictions", this.onRestrictions);
+        if (this.onUndo) this.$bus.unbind("undo", this.onUndo);

```

Comment:
Unbind in the reverse order from the binding. This pattern should always be applied in init/deinit scenarios, not only in Vue/JS applications but on every piece of software. The logic is: what is init'd afterward might depend on something that was init'd first, so they must be destroyed in reverse to avoid breaking dependencies.

Reply:
Got it !